### DISCUSSION TITLE
Display IDrawingLine in WinUi3?

### DISCUSSION THREAD

**Discussion by Petrarca181** [microsoft/microsoft-ui-xaml#8891] [Q&A]
Hello, How can I view in my desktop app List of IDrawingLine?

I use toolkit DrawingView in my MAUI project. List of lines is serialized to json. Than I want to view it in my winui project. 

ps. Yes I know maui is crossplatform. No, I don't want to use it in desktop apps.

**Comment by castorix** [comment]
To draw like with [DrawingView](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/drawingview), a way can be with the [InkPicture ](https://learn.microsoft.com/en-us/windows/win32/api/msinkaut/nn-msinkaut-iinkpicture)control :

![InkPicture](https://github.com/microsoft/microsoft-ui-xaml/assets/22345506/eaad91fd-cbd6-405e-b37c-746455737713)

**Comment by Petrarca181** [comment]
If somene will find this in the future

Direct usage for IDrawingLine not possible.  Use regex to get points from json. Convert points to lines than draw them on canvas.
